### PR TITLE
iotkit/helloworld: add instructions to build, deploy and run

### PR DESCRIPTION
**Pull request by rakyll**

### CODE REVIEW

File: iotkit/helloworld/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Wireless, by running the command below.
+
+    $ GOOS=linux GOARCH=arm GOARM=7 go build github.com/GoogleCloudPlatform/golang-samples/iotkit/helloworld
+
+The command below will generate a binary that can executable on your board.
+
+## Setting up the board
+
+Download [Debian 8.5 2016-05-13 4GB SD LXQT](https://beagleboard.org/latest-images)
+and write it to an microSD card.
```
Comment (broady) [inline iotkit/helloworld/README.md]: an -> a

File: iotkit/helloworld/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+First, go get the sample program.
+
+    $ go get -d github.com/GoogleCloudPlatform/golang-samples/iotkit/helloworld
+
+Then, build a binary targeting the GCP IoT Kit board, BeagleBone Green
+Wireless, by running the command below.
+
+    $ GOOS=linux GOARCH=arm GOARM=7 go build github.com/GoogleCloudPlatform/golang-samples/iotkit/helloworld
+
+The command below will generate a binary that can executable on your board.
```
Comment (broady) [inline iotkit/helloworld/README.md]: "that is executable" or "can be executed"

File: iotkit/helloworld/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+Join to the "BeagleBoneXXXX" network from your laptop and visit
+[192.168.8.1:3000/ide.html](192.168.8.1:3000/ide.html) to launch the
+Cloud9 IDE.
+
+Use "Files > Upload Local Files" to launch the uploader dialog.
+Drag and drop the "helloworld" binary and wait until the upload is finished.
+
+From the Cloud9 IDE terminal, run the following command the give the
+binary permissions to be executed.
```
Comment (broady) [inline iotkit/helloworld/README.md]: "run the following command to make the binary executable"

File: iotkit/helloworld/README.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Download [Debian 8.5 2016-05-13 4GB SD LXQT](https://beagleboard.org/latest-images)
+and write it to an microSD card.
+
+Put the microSD card into the SDcard slot and power the board to boot.
+Once the board boots, you will be able to see an unsecured Wifi network
+in the format of "BeagleBoneXXXX".
+
+Join to the "BeagleBoneXXXX" network from your laptop and visit
+[192.168.8.1:3000/ide.html](192.168.8.1:3000/ide.html) to launch the
```
Comment (broady) [inline iotkit/helloworld/README.md]: Add http:// to the URL. It didn't look linkable here:
https://github.com/GoogleCloudPlatform/golang-samples/blob/141b173893f1f7a855547d7ad0f1ea01488cc426/iotkit/helloworld/README.md

**Comment by rakyll** [comment]
All addressed, PTAL.

File: iotkit/helloworld/README.md [hunk 5]
```diff
@@ -17,26 +17,26 @@ Wireless, by running the command below.
 
     $ GOOS=linux GOARCH=arm GOARM=7 go build github.com/GoogleCloudPlatform/golang-samples/iotkit/helloworld
 
-The command below will generate a binary that can executable on your board.
+The command below will generate a binary that can be executable on your board.
```
Comment (broady) [inline iotkit/helloworld/README.md]: "can be executed", or "that is executable"

Comment (broady) [inline iotkit/helloworld/README.md]: Fixed it for you. LGTM

**Comment by rakyll** [comment]
@broady, is the CLA check failing because of your commit?

**Pull request merged**